content/browser/webui/brave_shared_resources_data_source.cc

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "brave/content/browser/webui/brave_shared_resources_data_source.h"
+
+#include <stddef.h>
+
+#include "base/feature_list.h"
+#include "base/files/file_path.h"
+#include "base/logging.h"
+#include "base/memory/ref_counted_memory.h"
+#include "base/strings/string_piece.h"
+#include "base/strings/string_util.h"
+#include "base/task/post_task.h"
+#include "build/build_config.h"
+#include "content/public/browser/browser_task_traits.h"
+#include "content/public/browser/browser_thread.h"
+#include "content/public/common/content_features.h"
+#include "content/public/common/url_constants.h"
+#include "ui/base/layout.h"
+#include "ui/base/resource/resource_bundle.h"
+#include "ui/base/webui/web_ui_util.h"
+#include "brave/ui/webui/resources/grit/brave_webui_resources.h"
+#include "brave/ui/webui/resources/grit/brave_webui_resources_map.h"
+
+#if defined(OS_WIN)
+#include "base/strings/utf_string_conversions.h"
+#endif
+
+namespace brave_content {
+
+using namespace content;
```

Comment:
This will fail lint, you should list them individually vs the entire module

Reply:
should also go above `namespace brave_content` which should just be `namespace brave`

Reply:
and also I'm not sure where `brave/content` came from, but this file should go in `brave/browser/webui`. `content` is a top-level module